onnxruntime/core/providers/shared_library/bridge_protobuf.h

```diff
@@ -0,0 +1,18 @@
+#include <string>
+
+#define SWIG
+namespace google {
+namespace protobuf {
+namespace internal {
+
+inline int
+ToIntSize(size_t size) {
+  return static_cast<int>(size);
```

Comment:
>return static_cast<int>(size); [](start = 2, length = 30)

The value of this function might be greater if in DEBUG builds it could check if the the value of size does not exceed std::numeric_limits<int>::max().